script.js

```diff
@@ -1,77 +1,81 @@
+//     paragraph2: 'loremLorem ipsum dolor sit amet consectetur adipisicing elit. Maxime mollitia',
+//     'mobile image': 'images.svg/Rectangle 34.svg',
+//     'desktop image': 'images.svg/Rectangle 34 (1).svg',
+//     technologies: ['HTML/CSS', 'RUBY ON RAILS', 'JAVASCRIPT'],
+//     liveVersion: '#',
+//     source: '#',
+//   },
+//   {
+//     id: 3,
+//     name: 'Project name gors here',
+//     paragraph1: 'loremLorem ipsum dolor sit amet consectetur adipisicing elit. Maxime mollitia',
+//     paragraph2: 'loremLorem ipsum dolor sit amet consectetur adipisicing elit. Maxime mollitia',
+//     'mobile image': 'images.svg/Rectangle 34.svg',
+//     'desktop image': 'images.svg/Rectangle 34 (1).svg',
+//     technologies: ['HTML/CSS', 'RUBY ON RAILS', 'JAVASCRIPT'],
+//     liveVersion: '#',
+//     source: '#',
+//   },
+//   {
+//     id: 4,
+//     name: 'Project name gors here',
+//     paragraph1: 'loremLorem ipsum dolor sit amet consectetur adipisicing elit. Maxime mollitia',
+//     paragraph2: 'loremLorem ipsum dolor sit amet consectetur adipisicing elit. Maxime mollitia',
+//     'mobile image': 'images.svg/Rectangle 34.svg',
+//     'desktop image': 'images.svg/Rectangle 34 (1).svg',
+//     technologies: ['HTML/CSS', 'RUBY ON RAILS', 'JAVASCRIPT'],
+//     liveVersion: '#',
+//     source: '#',
+//   },
+//   {
+//     id: 5,
+//     name: 'Project name gors here',
+//     paragraph1: 'loremLorem ipsum dolor sit amet consectetur adipisicing elit. Maxime mollitia',
+//     paragraph2: 'loremLorem ipsum dolor sit amet consectetur adipisicing elit. Maxime mollitia',
+//     'mobile image': 'images.svg/Rectangle 34.svg',
+//     'desktop image': 'images.svg/Rectangle 34 (1).svg',
+//     technologies: ['HTML/CSS', 'RUBY ON RAILS', 'JAVASCRIPT'],
+//     liveVersion: '#',
+//     source: '#',
+//   },
+//   {
+//     id: 6,
+//     name: 'Project name gors here',
+//     paragraph1: 'loremLorem ipsum dolor sit amet consectetur adipisicing elit. Maxime mollitia',
+//     paragraph2: 'loremLorem ipsum dolor sit amet consectetur adipisicing elit. Maxime mollitia',
+//     'mobile image': 'images.svg/Rectangle 34.svg',
+//     'desktop image': 'images.svg/Rectangle 34 (1).svg',
+//     technologies: ['HTML/CSS', 'RUBY ON RAILS', 'JAVASCRIPT'],
+//     liveVersion: '#',
+//     source: '#',
+//   },
+// ];
 const menu = document.querySelector('#menu-whole');
 const cancel = document.querySelector('#cancel');
 const menulist = document.querySelectorAll('.h-menu');
 const main = document.querySelector('main');
 const hamburger = document.querySelector('#hamburger');
 const mobileheader = document.querySelector('.mobile-header');
 const overlay = document.querySelector('#overlay');
-// const popupwindowDesktop = document.querySelector('#popup-window-desktop-container');
 // const seeprojectDesktop = document.querySelectorAll('.see-this-project');
```

Comment:
- [ ] As mentioned in the previous comment remove all commented out code from your code base